packages/ui-app/src/types.ts

```diff
@@ -5,6 +5,7 @@
 import { WithTranslation } from 'react-i18next';
 import { RpcRxInterface } from '@polkadot/rpc-rx/types';
 import { ActionStatus } from '@polkadot/ui-app/Status/types';
+import { LocationDescriptorObject } from 'history';
```

Comment:
This looks totally wrong. I wonder how to cleanly allow the `location` prop to have a `state`

Reply:
Not sure either. (And couldn't find something by a quick 2 min search). Maybe just comment, we are going to wonder about this 2 days from now...